board/fischertechnik/TXT/rootfs/opt/ftc/apps/system/display/display_de.ts

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <!DOCTYPE TS>
-<TS version="2.1" language="de" sourcelanguage="en">
+<TS version="2.0" language="de" sourcelanguage="en">
```

Comment:
Why is the version different here? And what should it be, 2.0 or 2.1?

Reply:
Really, I don´t know. This seems to be caused by linguist. Just ignore it